pkg/get/get_test.go

```diff
@@ -7739,3 +7739,56 @@ func Test_DownloadLazyDocker(t *testing.T) {
 	}
 
 }
+
+func Test_DownloadKeploy(t *testing.T) {
+	tools := MakeTools()
+	name := "keploy"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "v2.2.0-alpha19"
+
+	tests := []test{
+		{
+			os:      "linux",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     "https://github.com/keploy/keploy/releases/download/v2.2.0-alpha19/keploy_linux_amd64.tar.gz",
+		},
+		{
+			os:      "linux",
+			arch:    archARM64,
+			version: toolVersion,
+			url:     "https://github.com/keploy/keploy/releases/download/v2.2.0-alpha19/keploy_linux_arm64.tar.gz",
+		},
+		{
+			os:      "darwin",
+			arch:    archDarwinARM64,
+			version: toolVersion,
+			url:     "https://github.com/keploy/keploy/releases/download/v2.2.0-alpha19/keploy_darwin_all.tar.gz",
+		},
+		{
+			os:      "ming",
+			arch:    archARM64,
+			version: toolVersion,
+			url:     "https://github.com/keploy/keploy/releases/download/v2.2.0-alpha19/keploy_windows_arm64.tar.gz",
+		},
```

Comment:
what about intel based macs? (`arch64bit` + `darwin`)

Reply:
not yet supported, it's in long term roadmap